.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,47 +1,12 @@
-<!--
-  😀 Wonderful!  Thank you for opening a pull request for TypeORM.
+AB#XXXX <!--- Replace this with a reference to a devops issue -->
 
-  Please fill in the information below to expedite the review
-  and (hopefully) merge of your change.
+## Describe your changes
 
-  If unsure about something.. just do as best as you're able,
-  or reach out through our community support channels!
-  https://github.com/typeorm/typeorm/blob/master/docs/support.md
--->
+<!--- Add a brief description of your changes - not in-depth because the bulk of the description should be in the task on DevOps. -->
 
-### Description of change
+## Checklist before requesting a review
 
-<!--
-  Please be clear and concise what the change is intended to do,
-  why this change is needed, and how you've verified that it
-  corrects what you intended.
-
-  In some cases it may be helpful to include the current behavior
-  and the new behavior.
-
-  If the change is related to an open issue, you can link it here.
-  If you include `Fixes #0000` (replacing `0000` with the issue number)
-  when this is merged it will automatically mark the issue as fixed and
-  close it.
--->
-
-
-### Pull-Request Checklist
-
-<!--
-  Please make sure to review and check all of the following.
-
-  If an item is not applicable, you can add "N/A" to the end.
--->
-
-- [ ] Code is up-to-date with the `master` branch
-- [ ] `npm run format` to apply prettier formatting
-- [ ] `npm run test` passes with this change
-- [ ] This pull request links relevant issues as `Fixes #0000`
-- [ ] There are new or updated unit tests validating the change
-- [ ] Documentation has been updated to reflect this change
-- [ ] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)
-
-<!--
-  🎉 Thank you for contributing and making TypeORM even better!
--->
+- [ ] I have performed a self-review of my code
+- [ ] I have added tests wherever relevant
+- [ ] I have made sure that all automated checks pass before requesting a review
+- [ ] I do not need any deviation from our PR guidelines
```

Comment:
Do you want to link to the PR guidelines from 121 here? Or are you gonna duplicate them into this repository in a separate PR?